homeassistant/components/apcupsd/sensor.py

```diff
@@ -331,8 +336,8 @@
     ),
     "selftest": SensorEntityDescription(
         key="selftest",
-        name="UPS Last Self Test",
-        icon="mdi:calendar-clock",
+        name="UPS Self Test result",
```

Comment:
Changing the name of this entity is potentially a breaking change if the entity doesn't have a unique_id set. That seems to be a possibility for this integration.

Please add a breaking change paragraph explaining what changed and what the user needs to do in that case, ie updating automations and scripts that are referencing the entity_id.

Reply:
Done